pkg/intermodule/adaptor/account.ts

```diff
@@ -0,0 +1,76 @@
+import { Result } from '@mikuroxina/mini-fn';
+import type { Account, AccountID } from '../../accounts/model/account.js';
+import type { FetchService } from '../../accounts/service/fetch.js';
+import type { FetchFollowService } from '../../accounts/service/fetchFollow.js';
+import type {
+  AccountModuleInterface,
+  PartialAccount,
+} from '../interfaces/account.js';
+
+export class AccountModule implements AccountModuleInterface {
+  constructor(
+    private readonly fetchService: FetchService,
+    private readonly fetchFollowService: FetchFollowService,
+  ) {}
+
+  async fetchAccount(id: AccountID): Promise<Result.Result<Error, Account>> {
+    const res = await this.fetchService.fetchAccountByID(id);
+    if (Result.isErr(res)) {
+      return res;
+    }
+
+    return res;
+  }
+
+  async fetchFollowings(
+    id: AccountID,
+  ): Promise<Result.Result<Error, PartialAccount[]>> {
+    const res = await this.fetchFollowService.fetchFollowingsByID(id);
+    if (Result.isErr(res)) {
+      return res;
+    }
+
+    const accounts = await Promise.all(
+      res[1].map((v) => this.fetchService.fetchAccountByID(v.getTargetID())),
```

Comment:
```suggestion
      Result.unwrap(res).map((v) => this.fetchService.fetchAccountByID(v.getTargetID())),
```
`Result` の内部構造を直接触らないようにしたいです. 他の `res[1]` をしているコードも同様です.